_inc/lib/core-api/class.jetpack-core-api-site-endpoints.php

```diff
@@ -60,4 +66,160 @@ public static function get_features() {
 	public static function can_request() {
 		return current_user_can( 'jetpack_manage_modules' );
 	}
+
+
+	/**
+	 * Gets an array of data that show how Jetpack is currently being used to benefit the site.
+	 *
+	 * @since 7.7
+	 *
+	 * @return WP_REST_Response
+	 */
+	public static function get_benefits() {
+		$benefits = array();
+
+		/*
+		 * We get different benefits from Stats:
+		 * - this year's visitors
+		 * - Followers (only if subs module is active)
+		 * - Sharing counts (not currently supported in Jetpack -- https://github.com/Automattic/jetpack/issues/844 )
+		 */
+		$stats = null;
+		if ( function_exists( 'stats_get_from_restapi' ) ) {
+			$stats = stats_get_from_restapi( array( 'fields' => 'stats' ) );
+		}
+
+		// Yearly visitors.
+		if ( null !== $stats && $stats->stats->visitors > 0 ) {
+			$benefits[] = array(
+				'name'        => 'jetpack-stats',
+				'title'       => esc_html__( 'Jetpack Stats', 'jetpack' ),
```

Comment:
Do you see we need to mention that this is "Jetpack" stats? In other places in Jetpack, we call them "Site Stats".